### DISCUSSION TITLE
Avoid link to `/opt/homebrew/lib/`

### DISCUSSION THREAD

**Discussion by tillkruss** [Homebrew/discussions#3675] [Writing Formulae/Casks] [answered]
Hi!

We have a tap that supports installing several version of the same tap, depending on the PHP version: https://github.com/cachewerk/homebrew-tap

The tap seems to be linking various versions to the same `/opt/homebrew/lib/relay.so` file. How can we disable the link to `/opt/homebrew/lib/` entirely?

This seems to work (since we're linking to `/opt/homebrew/Cellar/` directly), but does unnecessary backups:

```ruby
link_overwrite "lib/relay.so"
```

Is the only viable approach to use dedicated names for this call?

```ruby
lib.install "relay.so"
```

**Comment by SMillerDev** [comment, marked as answer]
Make the other ones keg-only would be my suggestion

**Reply by tillkruss** [reply]
Thanks, that will do! ✌️
